tests/foreman/installer/test_installer.py

```diff
@@ -1584,3 +1585,79 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+def test_installer_inventory_plugin_update(destructive_sat):
```

Comment:
Do you think we need to add `@pytest.mark.destructive` any ways currently everything will be running un-grouped manner, having that marker would help in collection.